src/ansi-c/library/contracts.c

```diff
@@ -0,0 +1,59 @@
+#include <stdlib.h>
+
+/* FUNCTION: __CPROVER_is_fresh */
+#pragma GCC diagnostic ignored "-Wunused-parameter"
+_Bool __CPROVER_is_fresh(const void *ptr, __CPROVER_size_t size)
+{
+  __CPROVER_assert(
+    0, "__CPROVER_is_fresh called outside of contract function.");
+  return 0;
+}
+
+//
+// Below are templates of the functions that
+// will be instantiated for each contract function.
```

Comment:
Are these templates used?

I might be wrong, but it looks like we generate concrete instances of these templates during instrumentation, but from scratch.

Reply:
We're not using these templates at the moment, but generating concrete instances during instrumentation as you observed. I removed this for now, so we can make use of these templates on a separate PR (as discussed offline).